src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -349,23 +352,39 @@ export const CheckoutPayment = ({
       { setErrors }: FormikHelpers<QuoteInput>,
     ) => {
       try {
-        const { data } = await reCreateQuoteBundle(form)
-        const isUpdateQuotesFailed = isQuoteBundleError(data)
-        const limits = getLimitsHit(data)
-        if (isUpdateQuotesFailed && limits.length) {
-          setErrors({ ssn: textKeys.INVALID_FIELD() })
+        const { firstName, lastName, ssn, email, phoneNumber } = form
+        const allQuotes = getUniqueQuotesFromVariantList(bundleVariants)
+
+        for (const quote of allQuotes) {
+          await editQuote({
+            variables: {
+              quoteCartId,
+              quoteId: quote.id,
+              locale: locale.isoLocale,
+              payload: {
+                firstName,
+                lastName,
+                ssn,
+                email,
+                phoneNumber: phoneNumber?.replace(/\s/g, ''),
```

Comment:
Can be extracted as helper function

Reply:
Yeah, but in this case it seems kind of an over optimazation